pkg/util/clickhouse/clickhouse.go

```diff
@@ -32,19 +32,19 @@ const (
 	usernameKey         = "CLICKHOUSE_USERNAME"
 	passwordKey         = "CLICKHOUSE_PASSWORD"
 	urlKey              = "CLICKHOUSE_URL"
-	serviceName         = "clickhouse-clickhouse"
-	servicePortProtocal = "TCP"
+	ServiceName         = "clickhouse-clickhouse"
+	ServicePortProtocal = "TCP"
 	// #nosec G101: false positive triggered by variable name which includes "secret"
-	secretName = "clickhouse-secret"
+	SecretName = "clickhouse-secret"
 )
 
 var (
 	openSql         = sql.Open
 	createK8sClient = k8s.CreateK8sClient
 )
 
-func SetupConnection() (connect *sql.DB, err error) {
-	url, err := getClickHouseURL()
+func SetupConnection(client *kubernetes.Interface) (connect *sql.DB, err error) {
```

Comment:
```suggestion
func SetupConnection(client kubernetes.Interface) (connect *sql.DB, err error) {
```
Go interfaces are already pointers. Should avoid using pointers to interfaces

Reply:
Thanks Shawn for the reviewing! Updated.